#!/bin/bash
python Root_Cause\Data_Imbalance_Problem.py
